devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -467,6 +475,11 @@ def updated() {
 			sendEvent(name: 'status', value: 'closed', descriptionText: descriptionText, translatable: true)
 		}
 	}
+    
+	// Update configuration
+	configCmds += configure()
```

Comment:
Can you pull the setting of the acceleration threshold out into a separate function and just call that here rather than `configure`?

Reply:
Done Tom!